src/assets/js/tasklist.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable eqeqeq  */
+export default class Tasks {
+  constructor() {
+    this.list = localStorage.getItem('tasks')
+      ? JSON.parse(localStorage.getItem('tasks'))
+      : [];
+  }
+
+  removetask(indexval) {
+    this.list = this.list.filter((currentlist) => currentlist.index != indexval);
+    localStorage.setItem('tasks', JSON.stringify(this.list));
+  }
+
+  addtask(task) {
+    this.list.push(task);
+    localStorage.setItem('tasks', JSON.stringify(this.list));
+  }
+}
```

Comment:
- [ ] The function for editing tasks is missing, kindly implement it.I'd suggest adding an edit icon into each task and add an edit event listner to it(just like what you did with adding tasks).
- [ ] Users should be able to delete a completed task without clicking the `Clear All` button, I'd suggest adding a trash icon into each task and add a delete event listner to it(just like what you did with adding tasks).
- [ ] `localStorage` isn't updated properly when tasks are marked as completed(refershing the page will reset the progress), please implement this feature.